website/docs/provisioners/habitat.html.markdown

```diff
@@ -47,20 +47,25 @@ There are 2 configuration levels, `supervisor` and `service`.  Configuration pla
 
 ### Supervisor Arguments
 * `version (string)` - (Optional) The Habitat version to install on the remote machine.  If not specified, the latest available version is used.
-* `use_sudo (bool)` - (Optional) Use `sudo` when executing remote commands.  Required when the user specified in the `connection` block is not `root`.  (Defaults to `true`)
+* `license (string)` - (Optional) Sets the acceptance of Chef licensing: https://www.chef.io/end-user-license-agreement/
+* `auto_update (bool)` - (Optional) If set to `true`, the supervisor will auto-update itself as soon as new releases are available on the specified `channel`.
+* `http_disable (bool)` - (Optional) If set to `true`, disables the supervisor HTTP listener entirely.
+* `peers (array)` - (Optional) A list of IP or FQDN's of other supervisor instance(s) to peer with. (Defaults to none)
 * `service_type (string)` - (Optional) Method used to run the Habitat supervisor.  Valid options are `unmanaged` and `systemd`.  (Defaults to `systemd`)
 * `service_name (string)` - (Optional) The name of the Habitat supervisor service, if using an init system such as `systemd`. (Defaults to `hab-supervisor`)
-* `peer (string)` - (Optional) IP or FQDN of a supervisor instance to peer with. (Defaults to none)
```

Comment:
Removing this option/changing it to be `peers` and an array, feels like a disruptive change to existing users

Reply:
Yeah, I was a bit torn, as under the covers, you specify the peers with multiple `--peer` flags, at run time, like: `--peer 1.2.3.4 --peer 5.6.7.8 ...`.

Having it like that (as a string) requires the user to know the syntax of starting Habitat using `hab sup start <...>` (e.g., they must put in the literal string `--peer 1.2.3.4 --peer 5.6.7.8`), which I thought was a bit cumbersome, especially for new users.  I'd rather pass an list of items, and have the provisioner do the magic under the covers, but I dunno what Terraform recommends in cases like this.

Maybe I should allow both options, but put a strongly worded suggestion to use the `peers[]` list?  Or, I am fine with removing mine and sticking with `peer` as a string.

Reply:
I'm leaning a bit towards both options (to prevent "I upgraded, and everything broke!" grumpiness), I do agree that `peers` is a better option for many users, and would be nice to have.

Reply:
Okay, I'll look at re-adding the `peer` option to avoid upgrade grumpiness :smiley:.

Reply:
@pselle, after looking through the options a bit more, is there a way that I can specify a warning message to the user if `peer` is set that it's technically not recommended, and they should use `peers` instead, with a note that it (`peer`) may change/disappear in a future version?  I'm not sure what Terraform recommends in these cases, but it might be a nice cue to prompt the user with, if appropriate.

Reply:
I like that! That also helps out with educating people towards a more pleasant method. I'm not sure if it is, but if the `tfdiags` package can be used in a provisioner, that is our way of displaying similar messages in Terraform.

Reply:
@pselle, I couldn't figure out the `tfdiags` package, but I added the `peer` option back in, and updated the docs as well (including a small `deprecated` tag on the option in the docs).  Let me know what you think.

Reply:
Thanks! This works for me. I'll keep an eye on this, hoping to see if @mudash or @davymcaleer respond within the next couple of days, as I know they're interested in the license part.

Reply:
I might be late to the part but we do have attribute deprecation available:
https://www.terraform.io/docs/extend/best-practices/deprecations.html#provider-attribute-removal

These docs are always geared towards providers, but I think this should still apply to the provisioner schema